plumber/importer/__init__.py

```diff
@@ -26,6 +33,18 @@ class ImporterOperatorProps:
         options={"HIDDEN"},
     )
 
+    directory: StringProperty(
+        name="Directory",
+        maxlen=1024,
+        subtype="DIR_PATH",
+        options={"HIDDEN"},
+    )
+
+    files: CollectionProperty(
+        type=SelectedFile,
```

Comment:
The docs say to use this type:

files: bpy.props.CollectionProperty(type=bpy.types.OperatorFileListElement)

    When present in the operator this collection includes all selected files.


Use OperatorFileListElement instead of making our own type here if possible.